src/pudl/metadata/classes.py

```diff
@@ -1485,6 +1486,22 @@ def to_sql(
             constraints.append(key.to_sql())
         return sa.Table(self.name, metadata, *columns, *constraints)
 
+    def to_frictionless(self) -> frictionless.Resource:
+        """Convert to a Frictionless Resource."""
+        schema = frictionless.Schema(
+            fields=[
+                frictionless.Field(name=f.name, description=f.description)
+                for f in self.schema.fields
+            ],
+            primary_key=self.schema.primary_key,
+        )
```

Comment:
I believe the `type` values that we specify in our metadata are taken directly from the [Frictionless Table Schema spec](https://specs.frictionlessdata.io/table-schema/#types-and-formats), and so could also be output here, though only `name` is required by the spec.